_sass/_base.scss

```
@@ -155,17 +155,21 @@ p.blogdate {
 /*colors*/
 
 body {
-    font-family: Verdana, Arial, Helvetica, sans-serif;
-    font-size: 15px;
-    line-height: 1.7em;
+    font-family: Arial, Verdana, Helvetica, sans-serif;
+    font-size: 16px;
+    line-height: 1.6em;
```

```suggestion
    line-height: 1.5em;
```

did i say 1.6em? triying it over, i mean 1.5em :) with that, and the increased font-size, height and spacing are same as before (and aesthetic :), with, of course, the more compact font.